gendst/data/data.go

```diff
@@ -2135,6 +2157,20 @@ var Info = map[string][]Part{
 		Decoration{
 			Name: "Name",
 		},
+		Node{
+			Name:  "TypeParams",
+			Field: InnerField{"Type", "TypeParams"},
+			Type:  Struct{"FieldList"},
+		},
+		Decoration{
+			Name: "TypeParams",
+			Use:  Expr(func(n *jen.Statement) *jen.Statement { return n.Dot("Type").Dot("TypeParams").Op("!=").Nil() }),
+		},
+		SpecialDecoration{
```

Comment:
Was the use of `SpecialDecoration` intentional?

Reply:
It was. Do I misunderstand the use of the `SpecialDecoration` flag here? Like `Recv`, there is no reason to offer that decoration of `TypeParams` if there are no `TypeParams` defined. Again, please educate me -- you are the expert!

Reply:
Haha I'm a little hazy on how it works! I'd have to read through the code...

Reply:
If it was intentional you're probably correct.